scam/sane/hpaio.c: Include orblite.h for more function prototypes

Otherwise the build fails with compilers which do not support
implicit function declarations.

Submitted upstream: <https://bugs.launchpad.net/hplip/+bug/1997875>

diff --git a/scan/sane/hpaio.c b/scan/sane/hpaio.c
index d342626822fc2190..57d1ddea32fb5816 100644
--- a/scan/sane/hpaio.c
+++ b/scan/sane/hpaio.c
@@ -50,6 +50,7 @@
 #include "sclpml.h"
 #include "escl.h"
 #include "io.h"
+#include "orblite.h"
 #include "orblitei.h"
 
 
